$MirOS: ports/x11/icewm/patches/patch-lib_toolbar_in,v 1.1 2006/10/01 03:58:20 tg Exp $
--- lib/toolbar.in.orig	Sun Sep 10 16:11:48 2006
+++ lib/toolbar.in	Tue Dec 26 15:25:20 2006
@@ -4,6 +4,7 @@
 # since modifications to this file will be discarded when you
 # (re)install icewm.
 #
-prog XTerm xterm xterm
-prog FTE fte fte
-prog Netscape netscape netscape
+# $MirOS: ports/x11/icewm/patches/patch-lib_toolbar_in,v 1.1 2006/10/01 03:58:20 tg Exp $
+#
+prog UXTerm xterm uxterm
+prog Opera opera opera
